tests/flytekit/unit/clients/auth/test_token_client.py

```diff
@@ -71,8 +78,9 @@ def test_poll_token_endpoint(mock_requests):
     response = MagicMock()
     response.ok = True
     response.json.return_value = {"access_token": "abc", "expires_in": 60}
-    mock_requests.post.return_value = response
+    session.post.return_value = response
     r = DeviceCodeResponse(device_code="x", user_code="y", verification_uri="v", expires_in=1, interval=0)
     t, e = poll_token_endpoint(r, "test.com", "test", http_proxy_url="http://proxy:3000", verify=True)
-    assert t
-    assert e
+
+    assert t == "abc"
```

Comment:
Let's be more specific here. When adapting the magic mocks, I noticed that here one easily gets a `MagicMock` object for `t` which then also is not `None` but is not `"abc"`.